packages/__docs__/src/Nav/index.tsx

```diff
@@ -34,9 +33,11 @@ import { View } from '@instructure/ui-view'
 import { capitalizeFirstLetter } from '@instructure/ui-utils'
 
 import { NavToggle } from '../NavToggle'
-
-class Nav extends Component {
-  constructor(props) {
+import type { NavProps, NavState } from './props'
+class Nav extends Component<NavProps, NavState> {
```

Comment:
Please leave an empty line above the class declaration for better readability :)